views/file_list.erb

```diff
@@ -37,9 +39,16 @@
             <%= link_to_source_file(source_file) %>
           </td>
           <td class="<%= coverage_css_class(source_file.covered_percent) %> strong"><%= source_file.covered_percent.round(2).to_s %> %</td>
+          <% runtime_percentage = result.file_with_type(source_file, Coverband::RUNTIME_TYPE).try(:covered_percent).try(:round, 2) %>
```

Comment:
This looks much better. Not important but could move more logic into result object?

```erb
<%  = result.covered_percentage(source_file, Coverband::RUNTIME_TYPE) %>
```

Reply:
yeah I thought about that a bit, the problem is that the result object is pretty dumb in that it is just calling `method_missing`... I could make assumptions like the last argument is the type chop it off and use it to select the type and then call with the rest of the args, but that seemed a bit more complicated and magical, than the above approach.